source/process/sg_general-guidelines.rst

```diff
@@ -0,0 +1,92 @@
+
+The style guide applies to all end-user, developer, and administrator documentation that appears on the Mattermost documentation site at https://docs.mattermost.com.
+
+Apply the style guide in the following situations:
+
+- when you create a new document
+- when you convert a document from Markdown to reStructuredText
+- when you revise a document
+
+When you revise a document, apply the style guide rules to the part that you changed. If you have time to update the rest of the document, then do so. If updating the style for an entire document impacts a release, or if the scope is significantly greater than you originally anticipated, then do what you can and what makes sense, and then create an `issue in GitHub <https://github.com/mattermost/docs/issues>`_. Give the issue an appropriate title, such as *File XXX converted to .rst, but needs updating for style guide*.
+
+Write in the Context of Achievement
+-----------------------------------
+
+The documentation should help Mattermost users and administrators achieve their goals. Write imperative sentences as much as possible. Imperative sentences begin with verbs and give instructions, information, and advice to help people install, administer, and use Mattermost with success.
+
+Use positive constructs as much as possible, but note that a negative construct can act as a warning that causes a reader to pay closer attention to the content, resulting in higher levels of accomplishment.
+
+.. much of the following section comes from the Ubuntu Documentation Style Guide, which is in the public domain.
+Write to Facilitate Scanning
+----------------------------
+
+Readers need to find information quickly. People don't read documentation as much as they scan it for solutions to their immediate problem. Writing and presentation styles that seem redundant in essays or other texts are often helpful to people scanning for information.
+
+Use headings, lists, and emphasis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format text to give a visual hierarchy that allows readers to see the overall content of the page by scanning it. Use headings, lists, and emphasis to signal importance. Headings should summarize the topic of the underlying information so that when readers scan a page, they can get an accurate picture of the contents.
+
+Be clear
+  Write short, active sentences using everyday vocabulary. Maintain a visual separation between page elements.
+Be concise
+  Minimize content so it can be found and remembered. Keep pages short, modular, and focused on a single topic.
+Be consistent
+  Refer to one thing or idea with the same word throughout the page.
+
+Write for an International Audience
+-----------------------------------
+
+Mattermost documentation is translated into several languages for use in many countries and cultures. For this reason, you must consider cultural differences on a global scale. Names, places, events, and actions should be chosen as carefully as possible to avoid misunderstanding.
```

Comment:
Not really true yet. Only part of the end user docs are translated in product. But I understand the points here and it is helpful for even non-english speakers